plugins/coturn/coturn

```diff
@@ -0,0 +1,107 @@
+# -*- sh -*-
+
+set -e
+
+: << =cut
+
+=head1 NAME
+
+coturn - Simple plugin to monitor sessions on coturn server
+
+=head1 CONFIGURATION
+
+This plugin requires a password set in /etc/turnserver.conf for telnet 
+cli access. A telnet client has to be installed. Optionally you can 
+specify the port of the cli interface.
+
+[coturn]
+env.clipassword VERYSECUREPASSWORD
+env.port PORT
+
+=head1 AUTHOR
+
+Copyright (C) 2020 Sebastian L. <sebastiab@momou.ch>
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=manual
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+if [ -z ${port+x} ]; then cliport="5766"; else cliport=$port; fi
+
+get_coturn_stats() {
+	echo "open 127.0.0.1 $cliport"
+	sleep 1
+	echo $clipassword
+	sleep 1
+	for i in udp tcp dtls tls
+	do
+		echo "pu $i"
+		sleep 1
+	done
+}
+
+case $1 in
+    autoconf)
+        if [[ -x /usr/bin/turnserver ]]; then
+            echo yes
+            exit 0
+        else
+            echo "no (/usr/bin/turnserver not found)"
+            exit 0
+        fi
+        ;;
+   config)
+
+		echo "graph_title Sessions on coturn server"
+		echo "graph_vlabel turn sessions by protocols"
+		echo "graph_info This graph shows active sessions sorted by different protocols on the coturn server."
+		echo "graph_args --base 1000 --lower-limit 0"
+		echo "udp_sessions.label udp sessions"
```

Comment:
Maybe you want to use a stacked visualization for these connections? Simply add `.draw AREASTACK` for each field.